trusted_os/main.go

```diff
@@ -70,6 +79,33 @@ func init() {
 		if imx6ul.DCP != nil {
 			imx6ul.DCP.Init()
 		}
+
+		model, _ := usbarmory.Model()
+
+		switch model {
+		case usbarmory.BETA, usbarmory.GAMMA:
+			USB = usbarmory.USB1
+			USB.Init()
+
+			Control = usbarmory.USB2
+			Control.Init()
+
+			if debug {
+				debugConsole, _ := usbarmory.DetectDebugAccessory(250 * time.Millisecond)
+				<-debugConsole
+			}
+		case usbarmory.LAN:
+			LAN = usbarmory.ENET2
+			LAN.Init()
+
+			Control = usbarmory.USB1
+			Control.Init()
+		}
+
+		Storage = usbarmory.MMC
+	} else {
+		LAN = imx6ul.ENET1
+		LAN.Init()
```

Comment:
Which board type is being initialized here?

Reply:
This enters when we are not on Native CPU (see the if condition), so emulation.